GovUk.Frontend.ExampleApp/Views/Button/Index.cshtml

```diff
@@ -12,4 +12,15 @@
     <govuk-button class="govuk-button--secondary" type="button">Secondary button</govuk-button>
     <govuk-button class="govuk-button--warning" type="button">Warning button</govuk-button>
     <a href="/" class="govuk-link">Cancel</a>
-</div>
\ No newline at end of file
+</div>
+
+
+<govuk-notification-banner>
```

Comment:
Looking at these examples in the browser you have contrast issues for various states:

Hover:
<img width="754" alt="hover state" src="https://user-images.githubusercontent.com/1260550/219009011-9dc12d4d-a9bc-4184-b40e-78b485e60f8a.png">
Focus:
![focus state](https://user-images.githubusercontent.com/1260550/219009161-bcebedaa-0a69-4b8b-a872-9ef7d645f1bf.png)

Focus + active:
![focus + active](https://user-images.githubusercontent.com/1260550/219009378-57ff39c2-72c3-4742-8d7a-cdcf642ccc21.png)

The styles here should not be showing up differently to those in the Umbraco example, so clearly there is a difference between them and the styling needs to handle both. It may be because the Umbraco green banner uses a button group and this doesn't, but that suggests you also need to test the blue banner with a button group to see what happens there.

Reply:
These styles have been fixed to show either white or black text depending on the background colour and the behaviour is now consistent between the Umbraco and the Govuk example apps